client/src/partials/patients/record/patient_record.js

```diff
@@ -0,0 +1,25 @@
+angular.module('bhima.controllers')
+.controller('PatientRecordController', PatientRecordController);
+
+PatientRecordController.$inject = ['$stateParams', 'PatientService', 'NotifyService', 'moment'];
+
+function PatientRecordController($stateParams, Patients, Notify, moment) {
+  var vm = this;
+  var patientID = $stateParams.patientID;
+  
+  vm.loading = true;
+  /** @fixme if no uuid is provided this will download all the patients through the base url '/' */
+  Patients.read(patientID)
+    .then(function (result) {
+      vm.patient = result;
+      vm.loading = false;
+      
+      /** @todo move to service or mysql query */
+      vm.patient.name = [vm.patient.first_name, vm.patient.middle_name, vm.patient.last_name].join(' ');
```

Comment:
Interesting that this doesn't exist in our queries. It does in `Patients.list()` as `patientName`.  Since @mbayopanda will be updating the `Patients.detail()` route, to avoid conflicts we'll schedule this as a TODO.